lib/iris/tests/unit/cube/test_Cube.py

```diff
@@ -478,8 +503,15 @@ def test_string_coord_agg_by_label(self):
                                long_name='label', units='no_unit')
         self.assertEqual(res_cube.coord('val'), val_coord)
         self.assertEqual(res_cube.coord('label'), label_coord)
-
-    def test_string_coord_agg_by_val(self):
+        # For 2d coord, slices of aggregated coord should be the same as
```

Comment:
I'd like this 2d coord bit to be separated out into a different test please.

Reply:
@rcomer I'm not sure about written conventions, but I tend to take a look at the rest of the tests for guidance, and if that doesn't help I just use my instincts.